ui/main/src/app/modules/navbar/info/info.component.html

```diff
@@ -12,7 +12,7 @@
 </div>
 
 <div class="text-center navbar-info-block" style="margin-left: 5px;">
-    <div class="user-name" style="font-size:15px;text-align:right;">{{_userName$|async}}</div>
-    <div *ngIf="userEntitiesToDisplay"  class="user-entities" style="padding-top:10px;margin-left:5px;text-align:center;">{{userEntitiesToDisplay}}</div>
-    <div *ngIf="!userEntitiesToDisplay" class="user-entities" style="padding-top:5px;"> &nbsp;</div>
+    <div class="user-name" style="font-size:15px;text-align:right;padding-right:2em;">{{_userName$|async}}</div>
+    <div *ngIf="userEntitiesToDisplay"  class="user-entities" style="padding-top:10px;margin-left:5px;text-align:center; padding-right: 2em;">{{userEntitiesToDisplay}}</div>
+    <div *ngIf="!userEntitiesToDisplay" class="user-entities" style="padding-top:5px; padding-right: 2em;"> &nbsp;</div>
 </div>
```

Comment:
8px as padding is sufficient , the idea to have it as much as possible on the right